core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -668,7 +674,8 @@ case object ModelInsights {
         case p if p.param.name == OpPipelineStageParamsNames.InputFeatures =>
           p.param.name -> p.value.asInstanceOf[Array[TransientFeature]].map(_.toJsonString()).mkString(", ")
         case p if p.param.name != OpPipelineStageParamsNames.OutputMetadata &&
-          p.param.name != OpPipelineStageParamsNames.InputSchema => p.param.name -> p.value.toString
+          p.param.name != OpPipelineStageParamsNames.InputSchema =>
+          p.param.name -> Option(p.value).map(_.toString).getOrElse("null")
```

Comment:
not sure what we want here `"null"` seems like a weird choice ;) perhaps we should skip the nulls completely. @leahmcguire

Reply:
If this is to take care of cases when the parameter is unset then null makes sense...though I suppose we could flatmap and just not show those params

Reply:
I will omit the null params.